### DISCUSSION TITLE
Roadmap/Timeline for IEC 61508 certification

### DISCUSSION THREAD

**Discussion by finkephSICKAG** [zephyrproject-rtos/zephyr#83217] [General]
Hey everybody,

we would like to use Zephyr to develop products certified for IEC 61508. I found some information in the [Safety](https://docs.zephyrproject.org/latest/safety/index.html) section of the documentation, but I'm still curious about the roudmap/timeline when Zephyr will be certified for IEC 61508.

Is there already any date available when Zephyr has IEC 61508 certification?

**Comment by henrikbrixandersen** [comment]
CC: @simhein

**Comment by finkephSICKAG** [comment]
Happy new year everybody!

Are there any news about this topic? A rough estimation would help already, as we need this information for further planning.
